### PR TITLE
PS-2579 moar packages

### PR DESCRIPTION
**Pull request by pivnicek**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -15,25 +15,35 @@ RUN apt-get update \
         build-essential \
         curl \
         default-jdk \
+        default-jre \
```
Comment (odinuv) [inline Dockerfile]: isn't that part of jdk?

**Review by odinuv — APPROVED** [review]
is this going to be release for all? because if so, we might as well bump the base version to 4.0.5

File: Dockerfile (hunk 1 above)
Comment (pivnicek) [reply Dockerfile]: they've got it listed separately

Comment (pivnicek) [reply Dockerfile]: but yeah, I think you're right, it's included in the jdk

**Pull request merged**